src/sys/mod.rs

```diff
@@ -80,3 +80,60 @@ cfg_not_os_poll! {
         pub use self::unix::SourceFd;
     }
 }
+
+/// Define the `listen` backlog parameters as in the standard library. This
+/// helps avoid hardcoded unsynchronized values and allows better control of
+/// default values depending on the target.
+///
+/// Selecting a “valid” default value can be tricky due to:
+///
+/// - It often serves only as a hint and may be rounded, trimmed, or ignored by
+///   the OS.
+///
+/// - It is sometimes provided as a "magic" value, for example, -1. This
+///   value is undocumented and not standard, but it is often used to represent
+///   the largest possible backlog size. This happens due to signed/unsigned
+///   conversion and rounding to the upper bound performed by the OS.
+///
+/// - Default values vary depending on the OS and its version. Common defaults
+///   include: -1, 128, 1024, and 4096.
+///
+#[allow(dead_code)]
+pub(crate) const fn get_listen_backlog_size() -> i32 {
```

Comment:
Can this be a constant (not function)?

Also it shouldn't need `allow(dead_code)`